include/superior_mysqlpp/exceptions.hpp

```diff
@@ -25,19 +26,26 @@ namespace SuperiorMySqlpp
         using SuperiorMySqlppError::SuperiorMySqlppError;
 
         MysqlInternalError(const std::string& message,
-                           const char* mysqlError,
+                           const std::string& mysqlError_,
                            unsigned errorCode_)
-            : SuperiorMySqlppError(message + "\nMysql error: " + std::string(mysqlError))
+            : SuperiorMySqlppError(message + "\nMysql error: " + mysqlError_)
+            , mysqlError{mysqlError_}
             , errorCode{errorCode_}
         {
         }
 
+        StringView getMysqlError() const
+        {
+            return mysqlError;
+        }
+
         unsigned getErrorCode() const
         {
             return errorCode;
         }
 
     private:
+        const std::string mysqlError;
```

Comment:
I'd remove const here and on member below, it seems like a style antipattern without clear purpose.